pages/api/content/[type]/[slug].ts

```diff
@@ -126,6 +125,7 @@ const updateContent = async (req, res) => {
         })
     })
     .catch((err) => {
+      console.log('Invalid content')
```

Comment:
it should be done via logger

Reply:
sure, it was a remanent of fixing the tests